src/Utility/Xml.php

```diff
@@ -150,6 +150,8 @@ protected static function _loadXml($input, $options)
                 $xml = new DOMDocument();
                 $xml->loadXML($input);
             }
+        } catch (\Throwable $e) {
+            $xml = null;
         } catch (Exception $e) {
             $xml = null;
```

Comment:
**oh snap**

```
        if ($xml === null) {
            throw new XmlException('Xml cannot be read.');
        }
```

I did not see this before.

Ignore all my previous comments.